experiments/react-native-uifabric/src/components/Persona/Persona.helpers.ts

```diff
@@ -0,0 +1,66 @@
+import { PersonaSize } from '../PersonaCoin';
+import { ITextTokens } from '../../tokens';
+
+const textFontTable: { [key in PersonaSize]: ITextTokens } = {
+  size8: { fontSize: 'small' },
+  size24: { fontSize: 'medium' },
+  size32: { fontSize: 'medium' },
+  size40: { fontSize: 'medium' },
+  size48: { fontSize: 'medium' },
+  size56: { fontSize: 'xLarge' },
+  size72: { fontSize: 'xLarge' },
+  size100: { fontSize: 'xLarge' },
+  size120: { fontSize: 'xLarge' }
+};
+
+const secondaryFontTable: { [key in PersonaSize]: ITextTokens } = {
+  size8: { fontSize: 0 },

```

Comment:
What's the advantage to setting fontSize to 0 as opposed to null guarding in the caller?

Reply:
So I use this value to decide whether I should hide this text element. If a `fontSize` is undefined/null, it means the Text is rendered as the default font size.